app/components/log-drain-actions/template.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="panel-heading-actions">
+  {{#if logDrain.isProvisioned}}
+    <button class="btn btn-default btn-xs" {{action "restart" logDrain}}>Restart</button>
+  {{/if}}
+  {{#unless logDrain.hasBeenDeprovisioned}}
+    <button class="btn btn-default btn-xs" {{action "deprovision" logDrain}}>Deprovision</button>
+  {{/unless}}
```

Comment:
That nested logic really wasn't necessary -- thanks for that note. It actually allowed restarts of `pending` log drains, which wasn't intended.

It's now correctly showing the "Restart" action _only_ for `provisioned` log drains and the "Deprovision" action for anything other than `deprovisioning` (or `deprovisioned`) log drains.